examples/flow/stokesCavity.py

```diff
@@ -218,7 +218,7 @@
 
 >>> if __name__ == '__main__':
 ...     viewer = Viewer(vars=(pressure, xVelocity, yVelocity, velocity),
-...                xmin=0., xmax=1., ymin=0., ymax=1., colorbar=True)
+...                xmin=0., xmax=1., ymin=0., ymax=1., colorbar='vertical')
```

Comment:
This works for me.

Only thing that's weird for me is the velocity plot with the arrows. It's just a big black plot so too many arrows. That's irrelevant for this PR though.

Reply:
Too many arrows probably.

Reply:
`scale=5` backported from stokesNonNewtonian.ipynb